joomla/joomla-cms
# [com_checkin] Show only tables with check out items to check in

Pull Request for Improvement.
#### Summary of Changes

com_checkin behaves in a different way than the other components.
If show the tables even if there are no items to check in.

This PR changes that to only show the tables that have items to check in.
#### Testing Instructions
-  Use latest staging
- Open an article and don't save and/or close it (ex: change the url in the browser address bar to https://domain.tld/administrator/)
- Go to "System -> Global Check-in" you will see something like this:

![image](https://cloud.githubusercontent.com/assets/9630530/14412053/6134e952-ff51-11e5-9ff7-81cfbd3615ac.png)
- Apply patch with the patchtester
- Go again to "System -> Global Check-in" you will now see something like this:

![image](https://cloud.githubusercontent.com/assets/9630530/14412068/9bb24070-ff51-11e5-9740-cdcf0c077baa.png)
- Select that table and press "Check-in"
- You will now get a message like this

![image](https://cloud.githubusercontent.com/assets/9630530/14412075/bd733d40-ff51-11e5-8424-bfc2275e541e.png)
